Bug: "Cannot find module 'marked' or its corresponding type declarations." trying to use provided Trello import script

**Summary:**

Cloning main then following the directions at https://github.com/mattermost/focalboard/tree/main/import/trello results in four TypeScript errors:

```
~/focalboard/import/trello (main) $ npx ts-node importTrello.ts -i trello.json -o archive.focalboard

/home/me/focalboard/import/trello/node_modules/ts-node/src/index.ts:513
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
../../webapp/src/utils.ts:3:20 - error TS2307: Cannot find module 'marked' or its corresponding type declarations.

3 import marked from 'marked'
                     ~~~~~~~~
../../webapp/src/utils.ts:51:26 - error TS7006: Parameter 'href' implicitly has an 'any' type.

51         renderer.link = (href, title, contents) => `<a target="_blank" rel="noreferrer" href="${href}" title="${title || ''}" onclick="event.stopPropagation();">${contents}</a>`
                            ~~~~
../../webapp/src/utils.ts:51:32 - error TS7006: Parameter 'title' implicitly has an 'any' type.

51         renderer.link = (href, title, contents) => `<a target="_blank" rel="noreferrer" href="${href}" title="${title || ''}" onclick="event.stopPropagation();">${contents}</a>`
                                  ~~~~~
../../webapp/src/utils.ts:51:39 - error TS7006: Parameter 'contents' implicitly has an 'any' type.

51         renderer.link = (href, title, contents) => `<a target="_blank" rel="noreferrer" href="${href}" title="${title || ''}" onclick="event.stopPropagation();">${contents}</a>`
                                         ~~~~~~~~

    at createTSError (/home/me/focalboard/import/trello/node_modules/ts-node/src/index.ts:513:12)
    at reportTSError (/home/me/focalboard/import/trello/node_modules/ts-node/src/index.ts:517:19)
    at getOutput (/home/me/focalboard/import/trello/node_modules/ts-node/src/index.ts:752:36)
    at Object.compile (/home/me/focalboard/import/trello/node_modules/ts-node/src/index.ts:968:32)
    at Module.m._compile (/home/me/focalboard/import/trello/node_modules/ts-node/src/index.ts:1056:42)
    at Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Object.require.extensions.<computed> [as .ts] (/home/me/focalboard/import/trello/node_modules/ts-node/src/index.ts:1059:12)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
```

**Steps to reproduce the behavior:**
1. `git clone git@github.com:mattermost/focalboard.git`
2. `cd import/trello`
3. `npm i`
4. `cp ~/trello.json ./` (my exported file)
5. `npx ts-node importTrello.ts -i trello.json -o archive.focalboard`

**Expected behavior:**
An `archive.focalboard` matching what was in my `trello.json`.

**Platform:**
 - OS and version: Linux Mint 19.3
 - Node.js: v14.16.1
 - npm: v7.9.0
 - focalboard: fresh `git clone`, i.e.  https://github.com/mattermost/focalboard/commit/cae02a2ae222b0b1d2e2073da9405a88f6d3a542

**What doesn't fix it:**

- `npm i marked @types/marked` (same result, despite successful install)

**What does:**

- `cd ../../webapp`
- `npm i`

Note that if you _only_ do `npm i` in `webapp`, it also doesn't work.  So the instructions should say to do `npm i` in both `webapp` and `trello` before trying `npx ...`.

--- a/import/trello/README.md
+++ b/import/trello/README.md
@@ -3,9 +3,10 @@
 This node app converts a Trello json archive into a Focalboard archive. To use:
 1. From the Trello Board Menu, select `More`, then `Print and Export`, and `Export to JSON`
 2. Save it locally, e.g. to `trello.json`
-3. Run `npm install`
-4. Run `npx ts-node importTrello.ts -i <trello.json> -o archive.focalboard`
-5. In Focalboard, click `Settings`, then `Import archive` and select `archive.focalboard`
+3. Run `npm install` from within `focalboard/webapp`
+4. Run `npm install` from within `focalboard/import/trello`
+5. Run `npx ts-node importTrello.ts -i <path-to-trello.json> -o archive.focalboard` (also from within `focalboard/import/trello`)
+6. In Focalboard, click `Settings`, then `Import archive` and select `archive.focalboard`
 
 ## Import scope
 

